cx.ring.Ring.appdata.xml

```diff
@@ -0,0 +1,112 @@
+<!-- https://www.freedesktop.org/software/appstream/docs/chap-Metadata.html -->
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop">
+  <id>cx.ring.Ring</id>
+  <metadata_license>CC0-1.0</metadata_license>
+  <name>Ring</name>
+  <summary>Secure and distributed communication and SIP client</summary>
+  <description>
+    <p>
+      Ring is a secure and distributed file transfer, voice, video and chat communication platform 
+      that requires no centralized server and leaves the power of privacy in the hands of the user.
+    </p>
+    <p>
+      Ring also includes a SIP client.
+    </p>
+    <p>
+      Ring is free software which allows its users to communicate in multiple ways.
+    </p>
+    <ul>
+      <li>A telephone: a simple tool to connect, communicate and share.<br>
+      <li>A teleconferencing tool: easily join calls to create conferences with multiple 
+        participants.</li>
+      <li>A media sharing tool: Ring supports a variety of video input options, including 
+        multiple cameras and image and video files, and the selection of audio inputs 
+        and outputs; all this is supported by multiple high quality audio and video 
+        codecs.</li>
+      <li>A messenger: send text messages during calls or out of calls (as long as your 
+        peer is connected).</li>
+      <li>A building block for your IoT project: re-use the universal communications 
+        technology of Ring with its portable library on your system of choice.</li>
+    <ul>
+    <p>
+      Ring is also available on Windows, macOS, iOS and Android.
+    </p>
+    <p>
+      Ring a free and open-source cross-platform universal communication platform
+      developed by a team at Savoir-faire Linux and a growing community of contributors.
+      To to contribute, visit https://ring.cx/contribute
+    </p>
+  </description>
+  <!-- https://specifications.freedesktop.org/menu-spec/latest/apa.html -->
+  <!-- https://specifications.freedesktop.org/menu-spec/latest/apas02.html -->
+  <categories>
+    <category>Chat</category>
+    <category>Communication</category>
+    <category>InstantMessaging</category>
+    <category>Office</category>
+    <category>Productivity</category>
+  </categories>
+  <url type="homepage">https://ring.cx/</url>
+  <url type="bugtracker">https://ring.cx/contribute/bug-report</url>
+  <url type="faq">https://ring.cx/documentation/faq</url>
+  <url type="help">https://ring.cx/tutorials</url>
+  <!-- <url type="donation">…</url> -->
+  <url type="translate">https://ring.cx/contribute/translate</url>
+  <launchable type="desktop-id">cx.ring.Ring.desktop</launchable>
+ 
+  <!-- https://www.unixtimestamp.com/index.php -->
+  <releases>
+    <release date="2018-05-11" timestamp="1526155701" version="2018-05-12 20:08:21 UTC" />
+  </releases>
+ 
+  <!-- GNU and GNOME -->
+  <project_group>GNOME</project_group>
```

Comment:
It isn't a GNOME project afaik?

Reply:
It is a GNU project with a GNOME client. Do we change "GNOME" to "GNU"? Thank you

Reply:
GNU makes more sense.

Reply:
Updated PR to `GNU`. Did you see it?

Can you accept the PR to make it live?

Thank you